orc8r/cloud/Makefile

```diff
@@ -202,7 +204,9 @@ nms_prereqs_osx:
 	npm --version || brew install npm
 	yarn --version || npm install --global yarn
 	yarn add swagger-js-codegen
+	yarn add openapi-typescript-codegen
 
 nms_gen:
 	# generate Swagger API bindings for NMS
 	$(MAGMA_ROOT)/nms/scripts/generateAPIFromSwagger.sh $(SWAGGER_V1_YML) $(SWAGGER_NMS_OUT)
+	$(MAGMA_ROOT)/nms/scripts/generateAPIFromSwaggerForTypeScript.sh $(SWAGGER_V1_YML) $(SWAGGER_NMS_OUT_TS)
```

Comment:
I think it would be more idiomatic to call a `gen` target in the `nms` makefile. But since it was done that way also previously, it might be fine.